pkg/couchdb/couchdb.go

```diff
@@ -259,6 +259,10 @@ func dbNameHasPrefix(dbname, dbprefix string) (bool, string) {
 	return true, strings.Replace(dbname, dbprefix, "", 1)
 }
 
+func FindWithoutIndex(db Database, doctype, method, path string, reqBody interface{}, resBody interface{}) error {
```

Comment:
I don't where this function is used. And if you start a function by an upper-case letter, it means the function will be exported and can be used in other modules. For these functions, a comment is expected to explain what the function does (and the linter will insist on that).